apptuit/pyformance/apptuit_reporter.py

```diff
@@ -88,12 +100,21 @@ def __init__(self, registry=None, reporting_interval=10, token=None,
             else:
                 self.tags = {"host": socket.gethostname()}
         self.prefix = prefix if prefix is not None else ""
-        self.__decoded_metrics_cache = {}
-        self.client = Apptuit(token, api_endpoint, ignore_environ_tags=True)
+        self.client = Apptuit(token=token, api_endpoint=api_endpoint,
+                              ignore_environ_tags=True, sanitize_mode=sanitize_mode)
         self._meta_metrics_registry = MetricsRegistry()
         self.error_handler = error_handler
+        self.pid = os.getpid()
+        self.process_metrics = None
+        if collect_process_metrics:
+            self.process_metrics = ProcessMetrics(self.pid, self.registry)
```

Comment:
just call os.getpid() inside the ProcessMetrics, if we don't need it in the reporter for any other purpose